services/core-web/src/components/mine/Projects/Project.js

```diff
@@ -45,6 +45,16 @@ export class Project extends Component {
     this.handleScroll();
   }
 
+  // Handle clicking an activity notification link(MMA tab) while Project component is mounted
+  componentDidUpdate(prevProps) {
+    const currentTab = this.props.match.params.tab;
```

Comment:
is this specific to the project page in that the link from the notification drawer wasn't working?

Reply:
What was happening is the URL gets updated to the correct one using the link however since this page is in a tab the activeTab(local state) never gets refreshed since the component is still mounted. I'm not super happy doing it this way but no other clean options presented themselves.